src/udiskssimplejob.c

```diff
@@ -109,8 +109,10 @@ udisks_simple_job_complete (UDisksSimpleJob     *job,
                             gboolean             success,
                             const gchar         *message)
 {
+  static const gchar none[] = "";
+
   g_return_if_fail (UDISKS_IS_SIMPLE_JOB (job));
-  udisks_job_emit_completed (UDISKS_JOB (job), success, message);
+  udisks_job_emit_completed (UDISKS_JOB (job), success, (message) ? message : none);
```

Comment:
When I did the original change it was before I knew what the needed lifetime of the string needed to be, thus I made it static to ensure it would exist if beyond the lifetime of the function invocation.  However, after I found that many others are simply using a string constant as well I could of gone back and changed to what you suggested.

I'll redo.